web/pm.html

```diff
@@ -0,0 +1,299 @@
+      alt="Veritas Logo" class="img-fluid veritas-logo">
+    <h1 class="text-center" style="display: inline-block;"> NetBackup FlexScale Manager </h1>
+    <span class="text-left" style="margin-left: 80%;">
+      <input type="checkbox" id="developer-mode" onclick="updateForm(this);">
+      <label class="text-left" for="download">Developer mode</label> <br>
+    </span>
+  </header>
+  <form>
+    <div class="h-full grid"
+      style="grid-template-columns: 0.955224fr 10px 1.04478fr; grid-template-rows: 1.2521fr 10px 0.747899fr;">
+      <div>
+        <label for="env">
+          <h2>README</h2>
+          <ol>
+            <li>Select one or more action checkboxes on the left pane</li>
+            <li>Enter parameters related to selected actions in textboxes on the right pane</li>
+            <li>Specify any additional parameters in the textarea one per line in the format: <b
+                style="color: red;">Key=Value</b>
+            </li>
+            <li>View steps or start running</li>
+          </ol>
+        </label><br> </p>
+
+        <h2>Actions</h2>
+        <table class="table table-sm caption-top">
+          <caption>1. Select one or more action checkboxes...</caption>
+          <tr>
+            <!-- <th><h3>Actions</h3></th> -->
+          </tr>
+
+          <tr>
+            <td>
+              <h4>Download</h4>
+            </td>
+          </tr>
```

Comment:
Seems like an odd use of a table when every row has exactly one column. What's _tabular_ about this?

Reply:
I'm not at all an UI expert... but while trying out multiple things... I ended up liking this one...

![image](https://github.com/user-attachments/assets/3121917c-0b6b-4f40-9562-66654cf4dc92)